test/docgen/view/initializer.test.ts

```diff
@@ -28,3 +28,42 @@ test.each(Language.values())('%s snapshot', (language) => {
   expect(init).toMatchSnapshot();
   expect(markdown.visitInitializer(init).render()).toMatchSnapshot();
 });
+
+const variadicTestAssembly: reflect.Assembly = Assemblies.instance.variadicExampleTest;
+const findVariadicInitializer = (): reflect.Initializer => {
+  for (const klass of variadicTestAssembly.classes) {
+    if (klass.initializer) {
+      return klass.initializer;
+    }
+  }
+  throw new Error('Assembly does not contain an initializer');
+};
+
+const findVariadicParameter = (): reflect.Parameter => {
+  for (const klass of variadicTestAssembly.classes) {
+    if (klass.initializer) {
+      for (const param of klass.initializer.parameters) {
+        if (param.variadic) {
+          return param;
+        }
+      }
+    }
+  }
+  throw new Error('Assembly does not contain a parameter');
+};
+
+test('variadic parameter in initializer is correct', () => {
```

Comment:
You're testing here that a test function behaves like you expect. Tests shouldn't be testing tests :)

If the `findVariadicParameter` test is incorrect, than the snapshot would be incorrect, and you should iterate until it is.